spec/system/jobseekers/jobseeker_applications_statuses_spec.rb

```diff
@@ -49,84 +49,5 @@
         expect(page).to have_css("#employment_history", text: I18n.t("shared.status_tags.incomplete"))
       end
     end
-
-    context "when the jobseeker has completed some details in their profile but not all of them" do
-      let!(:jobseeker_profile) { create(:jobseeker_profile, :with_qualifications, :with_employment_history, jobseeker: jobseeker, qualified_teacher_status: nil) }
-
-      it "shows all sections that have been filled in with the status tag 'in progress' and shows empty sections with the status tag 'not started'" do
```

Comment:
This was not testing anything new, they all just show up as incomplete now whenever they are imported